doc/manuals/user/walkthrough_apiv2.md

```diff
@@ -738,9 +734,8 @@ Write it down because you will need it later in this tutorial.
 < Date: Wed, 25 May 2016 11:05:35 GMT
 ```
 
-Let's have a look now at accumulator-server.py. We will see one (and
-just one by the moment, no matter how much you wait)
-notifyContextRequest, similar to this one:
+Let's have a look now at accumulator-server.py. We will see one notifyContextRequest (and
```

Comment:
We will see one notification (and ...

(notifyContextRequest is an old term from NGSIv1. This comment is not really related with fowareding functionality, but given you have touched this part of the walkthrough document, let's ensure is as correct as possible :)

Reply:
Fixed in aef7cb8